src/main/webapp/script.js

```diff
@@ -19,41 +19,47 @@ function getRecipeInfo() {
   params.append('image', image);
   const request = new Request('/dishAnalysis', {method: "POST", body: params});
   fetch(request).then(response => response.json()).then((recipeListInfoJson) => {
-    sessionStorage.recipeList = JSON.parse(recipeListInfoJson);
-    window.location.href = "/display.html";
+    //set top options to radio buttons
```

Comment:
Instead of adding explicit 2 options, do you consider making it more flexible for to allow the potential change of the top size? e.g. you might want to have top 3 options later

And nit: one more space after "//" and

Reply:
Right now the idea is just for two options, maybe the idea for additional options would be a good stretch goal